frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -1591,9 +1588,9 @@ test("typeahead_results", () => {
     assert_stream_matches("denmark", [denmark_stream]);
     assert_stream_matches("denmark ", []);
     assert_stream_matches("den ", []);
-    assert_stream_matches("cold", [sweden_stream, denmark_stream]);
+    // assert_stream_matches("cold", [sweden_stream, denmark_stream]);
```

Comment:
We generally don't leave commented out things in tests like this; they are bad for readability. I think it'd be better to just have these lines be things like: 

`assert_stream_matches("cold", [])`

With a comment noting that we don't match against descriptions, for all the test changes in this file.